### PR TITLE
Traversal and Collections docs

### PR DESCRIPTION
**Pull request by JR-Morgan**
Added Traversal docs to their own page.
Added Collections docs to "The Base Object"

Compared with the draft on notion, I've rephrased some of the first couple paragraphs

### CODE REVIEW

File: .vuepress/config.js [hunk 1]
```diff
@@ -183,6 +183,7 @@ module.exports = {
             "connectors-dev",
             "kits-dev",
             "transports-dev",
+            "traversal",
```
Comment (jsdbroughton) [inline .vuepress/config.js:186]: URL is traversal; the section is titled "Consuming Structured Data" - I get it, but will everyone?

File: dev/base.md [hunk 2]
```diff
@@ -157,3 +157,44 @@ For example, When receiving **BIM types** like `Wall`, `Floor`, `Beam`, etc into
...
+
+
+---
+
+Our new front-end (fe2) is designed to display `Collection` in the scene explorer.
+
+![Collections in frontend 2](../dev/img/core/fe2-collections.gif)
+
+Suppose we look at how Collection objects are used inside Speckle Commits (Versions).
+We see within the Collection’s `elements` property; we can expect any objects to be nested, including sub-collections.
```
Comment (jsdbroughton) [inline dev/base.md:186]: I'd make mention of `elements` (should) being always detached, but we haven't introduced detaching yet in the concepts.

Comment (JR-Morgan) [reply dev/base.md:186]: `Collection.elements` is typed as a detached property.
But do you mean, in general, we should document that "elements" should be detached? I agree, just not sure the best place for it.

Comment (jsdbroughton) [reply dev/base.md:186]: I think we can tie it together in the detaching doc. 

e.g.
"_For this reason, the `elements` property is detached_"

**Pull request merged**